LEGO1/lego/sources/misc/legowegedge.h

```diff
@@ -0,0 +1,28 @@
+#ifndef __LEGOWEGEDGE_H
+#define __LEGOWEGEDGE_H
+
+#include "legoweedge.h"
+#include "mxgeometry/mxgeometry3d.h"
+#include "mxtypes.h"
+
+// SIZE 0x54
+class LegoWEGEdge : public LegoWEEdge {
```

Comment:
Lacks `VTABLE` annotation (and possibly stubs for virtual functions if there are any)